src/components/DialogBox/styles.scss

```diff
@@ -17,6 +15,18 @@
   z-index: 1300;
   inset: 0px;
 
+  &--skrim {
+    background-color: rgba($tw-clr-base-gray-025, $tw-opcty-base-060);
+    background-image: linear-gradient(180deg, rgba($tw-clr-base-green-050, $tw-opcty-base-000), rgba($tw-clr-base-green-050, 0.24));
```

Comment:
I was told by @hannah-cai that this was how I should construct the background gradient. Namely, a solid `tw-clr-base-gray-025` overlaid by a `tw-clr-base-green-050` gradient from transparent to 40% (for the skrim, the opacities were multiplied by 0.6 per the spec).

Reply:
👍 
@charlie-tf can we loop @chudo76 in here... using these base tokens is no different from using hardcoded colors, etc.

Reply:
@chudo76 and I have agreed that not every use of color needs to have a semantic token, we will be using base colors occasionally for things that are more or less one-offs, and which don't have a semantic meaning

Reply:
Thanks for clarifying @hannah-cai , sounds good 👍